content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -112,6 +173,19 @@ ll euclid(ll a, ll b) {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+
+def euclid(a: int, b: int) -> int:
+    """Recursive Euclidean GCD."""
+    if not b:
+        return a
+    return euclid(b, a % b)
```

Comment:
a ternary would work too (optional)

Reply:
Do you mean something like:

```py
return a if b else euclid(b, a % b)
```

That would work, but I think keeping it slightly more verbose might be better for a tutorial/reference website.

Reply:
fair enough